action.yml

```diff
@@ -107,42 +107,42 @@ runs:
-          MAVEN_SNAPSHOT_SERVER_PASSWORD="${{ inputs.mavenSnapshotServerPassword }}"
-          MAVEN_RELEASE_EXECUTE_PERFORM="${{ inputs.mavenReleaseExecutePerform }}"
-          MAVEN_RELEASE_PUSH_COMMITS="${{ inputs.mavenReleasePushCommits }}"
-          MAVEN_RELEASE_EXECUTE_POST_SCRIPTS="${{ inputs.mavenReleaseExecutePostScripts }}"
-          MAVEN_RELEASE_EXECUTE_PRE_SCRIPTS="${{ inputs.mavenReleaseExecutePreScripts }}"
-          MAVEN_RELEASE_ARGUMENTS="${{ inputs.mavenReleaseArguments }}"
-          GIT_USER_NAME="${{ inputs.gitUserName }}"
-          GIT_USER_EMAIL="${{ inputs.gitUserEmail }}"
-          MAVEN_RELEASE_TAG="${{ inputs.mavenReleaseTag }}"
-          MAVEN_RELEASE_VERSION="${{ inputs.mavenReleaseVersion }}"
-          MAVEN_NEXT_DEVELOPMENT_VERSION="${{ inputs.mavenNextDevelopmentVersion }}"
+        RELEASE_POST_SCRIPTS=${{ inputs.mavenPostScripts }}
+        RELEASE_PRE_SCRIPTS=${{ inputs.mavenPreScripts }}
+        MAVEN_BIN=${{ inputs.mavenBinary }}
+        MAVEN_PROJECT_POM=${{ inputs.mavenProjectPom }}
+        RELEASE_BRANCH_NAME=${{ inputs.releaseBranchName }}
+        MAVEN_RELEASE_SETTINGS_XML=${{ inputs.mavenReleaseSettingsXml }}
+        MAVEN_RELEASE_SERVER_ID=${{ inputs.mavenReleaseServerId }}
+        MAVEN_RELEASE_SERVER_USERNAME=${{ inputs.mavenReleaseServerUsername }}
+        MAVEN_RELEASE_SERVER_PASSWORD=${{ inputs.mavenReleaseServerPassword }}
+        MAVEN_SNAPSHOT_SERVER_ID=${{ inputs.mavenSnapshotServerId }}
+        MAVEN_SNAPSHOT_SERVER_USERNAME=${{ inputs.mavenSnapshotServerUsername }}
+        MAVEN_SNAPSHOT_SERVER_PASSWORD=${{ inputs.mavenSnapshotServerPassword }}
+        MAVEN_RELEASE_EXECUTE_PERFORM=${{ inputs.mavenReleaseExecutePerform }}
+        MAVEN_RELEASE_PUSH_COMMITS=${{ inputs.mavenReleasePushCommits }}
+        MAVEN_RELEASE_EXECUTE_POST_SCRIPTS=${{ inputs.mavenReleaseExecutePostScripts }}
+        MAVEN_RELEASE_EXECUTE_PRE_SCRIPTS=${{ inputs.mavenReleaseExecutePreScripts }}
+        MAVEN_RELEASE_ARGUMENTS=${{ inputs.mavenReleaseArguments }}
+        GIT_USER_NAME=${{ inputs.gitUserName }}
+        GIT_USER_EMAIL=${{ inputs.gitUserEmail }}
+        MAVEN_RELEASE_TAG=${{ inputs.mavenReleaseTag }}
+        MAVEN_RELEASE_VERSION=${{ inputs.mavenReleaseVersion }}
+        MAVEN_NEXT_DEVELOPMENT_VERSION=${{ inputs.mavenNextDevelopmentVersion }}
         EOF
     - name: "Set GIT automation credentials"
       run: git config --global user.name "${{ inputs.gitUserName }}" && git config --global user.email "${{ inputs.gitUserEmail }}"
       shell: bash
+    - name: "Export release version"
+      id: version
+      run: ${{ github.action_path }}/version.sh
+      shell: bash
     - name: "Check release prerequisites"
       run: ${{ github.action_path }}/check.sh
       shell: bash
     - name: "Run pre actions"
       run: ${{ github.action_path }}/pre.sh ${{ inputs.mavenReleaseArguments }}
       shell: bash
-    - name: "Export release version"
```

Comment:
Oh this is just a move :)

Reply:
Yeah, that's required for pre actions to be able to read version